src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_DeconstructionAssignmentOperator.cs

```diff
@@ -54,6 +54,7 @@ public override BoundNode VisitDeconstructionAssignmentOperator(BoundDeconstruct
                 stores.Add(assignment);
             }
 
+            stores.Add(new BoundVoid(node.Syntax, node.Type));
             var result = _factory.Sequence(temps.ToImmutable(), stores.ToArray());
```

Comment:
Yes, found it. There is an overload that takes a separate BoundExpression for the result. I'll use that. Thanks